write.go

```diff
@@ -418,10 +428,6 @@ func writePixelData(w dicomio.Writer, t tag.Tag, value Value, vr string, vl uint
 		numValues := len(image.Frames[0].NativeData.Data[0])
 		length := numFrames * numPixels * numValues * image.Frames[0].NativeData.BitsPerSample / 8 // length in bytes
 
-		err := encodeElementHeader(w, t, vr, uint32(length))
-		if err != nil {
-			return err
-		}
```

Comment:
don't we still need to encode the element header in the else block?

Reply:
Based on my testing and understanding so far, I don't think we need to encode the element header in the if or the else blocks since the element header gets written at line 181 in the writeElement function. I found that the first 8 bytes of pixel data was a duplicate element header because of the redundancy